syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -992,19 +992,57 @@ def torch_type(self):
         else:
             return self.child.torch_type()
 
-    def encrypt(self, public_key):
-        """This method will encrypt each value in the tensor using Paillier
-        homomorphic encryption.
+    def encrypt(self, encryption_method="mpc", **kwargs):
```

Comment:
I have a slight preference for `protocol` instead of `encryption_method` (mostly because its shorter and because we often speak of crypto protocols)